lib/Client.js

```diff
@@ -7,9 +7,22 @@ var min = require('osc-min');
 var Message = require('./Message');
 
 var Client = function (host, port) {
+    var isBroadcast = host.split('.').reduce(function(previous, current){
+        return previous ? true : +current === 255;
+    }, false),
+    setter = function () {
+        this.setBroadcast(isBroadcast);
```

Comment:
Hi, sorry for the late reply, I am a bit busy at the moment but will get to it as soon as possible.

I probably gave you a bit too little context: this is a very crude implementation for the prototype I have been working on. I just wanted to make sure this doesnt get lost. Multicast is not implemented at all, even broadcasting is implemented rather buggy as you noted.